app/src/main/java/com/example/pygmyhippo/user/EventFragment.java

```diff
@@ -178,6 +157,14 @@ public void OnCompleteDB(@NonNull ArrayList<Event> docs, int queryID, int flags)
                 event = docs.get(0);
                 populateTextFields();
             }
+            else {
+                // TODO: popback here
```

Comment:
I wonder if it makes sense to have the logic for this here, and not in qr scanner